core/scripts/ocr2vrf/util.go

```diff
@@ -342,7 +342,7 @@ func setBeaconBilling(e helpers.Environment, vrfBeaconAddr string, maximumGasPri
 }
 
 func setCoordinatorBilling(e helpers.Environment, vrfCoordinatorAddr string, useReasonableGasPrice bool, unusedGasPenaltyPercent uint8,
-	stalenessSeconds, redeemableRequestGasOverhead, callbackRequestGasOverhead, premiumPercentage, reasonableGasPriceStalenessBlocks uint32,
+	stalenessSeconds, redeemableRequestGasOverhead, callbackRequestGasOverhead, reasonableGasPriceStalenessBlocks uint32, premiumPercentage uint8,
 	fallbackWeiPerUnitLink *big.Int) {
 	coordinator := newVRFCoordinator(common.HexToAddress(vrfCoordinatorAddr), e.Ec)
```

Comment:
Can't comment on that line but there's an unnecessary cast `uint8(premiumPercentage)` below.

Reply:
oh right. fixed it